notebooks/06_Serving/06_01_Seldon_Core_Serving.ipynb

```diff
@@ -0,0 +1,185 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Seldon Core Serving Example\n",
+    "\n",
+    "Seldon Core comes installed with Kubeflow. The [Seldon Core documentation site](https://docs.seldon.io/projects/seldon-core/en/latest/) provides full documentation for running Seldon Core inference.\n",
+    "\n",
+    "Seldon Core is an open source platform to deploy your machine learning models on Kubernetes at massive scale.It converts ML models (Tensorflow, Pytorch, H2o, etc.) or language wrappers (Python, Java, etc.) into production REST/GRPC microservices.\n",
+    "\n",
+    "Seldon handles scaling to thousands of production machine learning models and provides advanced ML capabilities out of the box including Advanced Metrics, Request Logging, Explainers, Outlier Detectors, A/B Tests, Canaries and more."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Note: \n",
+    "Since you're in anonymous namespace lack of permission to create other namespaces and perform operations on that namespace, you should copy all the yaml files and execute all of the below code blocks in your local environment. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Create New Namespace"
```

Comment:
Do we need step 1-3?  Ideally, in multi-user environment, user may not have permission to create namespace.

Reply:
We need to label namespace with `serving.kubeflow.org/inferenceservice=enabled`, so we can run inference task in it. For the rest of the steps, we don't need them.

Reply:
I remember this is for kfserving only? Does seldon need it?

Reply:
It's also for seldon serving. Reference from here (https://www.kubeflow.org/docs/components/serving/seldon/#simple-example)